src/shared/components/create-case-filters/create-case-filters.component.spec.ts

```diff
@@ -262,6 +296,8 @@ describe('CreateCaseFiltersComponent', () => {
     mockAlertService = createSpyObj<AlertService>('alertService', ['clear']);
     mockDefinitionsService = createSpyObj('mockDefinitionsService', ['getJurisdictions']);
     mockDefinitionsService.getJurisdictions.and.returnValue(Observable.of([JURISDICTION_2]));
+    sessionStorageService = jasmine.createSpyObj<SessionStorageService>('sessionStorageService', ['getItem']);
+    sessionStorageService.getItem.and.returnValue(`{"id": 1, "forename": "Firstname", "surname": "Surname", "roles": ["role1", "role3"], "email": "test@mail.com","token": null}`)
```

Comment:
This is comfortably more than 140 characters long. I would expect it to fail linting... and if it doesn't do it yet, it will as soon as the linting is standardised across all the projects.

Reply:
adjusted on next line